prow/plank/controller_test.go

```diff
@@ -1335,7 +1337,7 @@ func TestSyncPendingJob(t *testing.T) {
 			ExpectedURL:      "homeless/error",
 		},
 		{
-			Name: "scheduled, pending started more than podUnscheduledTimeout ago",
```

Comment:
While the old testname correctly described the situatation, the podScheduledTimeout is completely irrelevant once the pod was scheduled so I have changed it to clarify